fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -52,23 +53,24 @@ object FAPI {
       configJson <- Response.Async.Right(faciaClient.config)
       frontJson <- Response.fromOption(configJson.fronts.get(frontId), NotFound(s"No front found for $frontId"))
       collectionIds = frontJson.collections
-      collectionsJsons <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsonOptions <- Response.Async.Right(Future.traverse(collectionIds)(faciaClient.collection))
+      collectionsJsons = collectionsJsonOptions.flatten
       collectionConfigJsons <- Response.traverse(
         collectionIds.map(id => Response.fromOption(configJson.collections.get(id), NotFound(s"Collection config not found for $id")))
       )
       collectionConfigs = collectionConfigJsons.map(CollectionConfig.fromCollectionJson)
     } yield {
       (collectionIds, collectionsJsons, collectionConfigs).zipped.toList.map { case (collectionId, collectionJson, collectionConfig) =>
-        Collection.fromCollectionJsonConfigAndContent(CollectionId(collectionId), collectionJson, collectionConfig)
+        Collection.fromCollectionJsonConfigAndContent(collectionId, collectionJson, collectionConfig)
       }
     }
   }
 
   def collectionContent(collection: Collection, adjustSearchQuery: AdjustSearchQuery = identity)
                        (implicit capiClient: GuardianContentClient, ec: ExecutionContext): Response[List[CuratedContent]] = {
-    val itemIds = collection.live.map(_.id)
+    val itemIdsForRequest = collection.live.filterNot(_.isSnap).map(_.id)
```

Comment:
This logic should probably be elsewhere if possible. Better to keep logic out of the FAPI file because it can't be tested here and I think this is something that needs a test (if only to prevent regressions).